tests/openedx_tagging/core/tagging/test_system_defined_models.py

```diff
@@ -207,37 +252,25 @@ class TestLanguageTaxonomy(TestTagTaxonomyMixin, TestCase):
     Test for Language taxonomy
     """
 
-    # FIXME: something is wrong with this test case. It's setting the string
```

Comment:
@bradenmacdonald  Same of [this comment](https://github.com/openedx/openedx-learning/pull/87/files#r1343052892)